tools/src/test/scala/scala/scalanative/linker/TraitReachabilitySuite.scala

```diff
@@ -30,7 +30,8 @@ class TraitReachabilitySuite extends ReachabilitySuite {
   val ObjectInit: Global = g("java.lang.Object", Sig.Ctor(Seq.empty))
   val Test: Global = g("Test$")
```

Comment:
Should you refactor out `val TestClass = "Test$"` like you did above for `MainClass`?